target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -304,6 +304,7 @@ ath79_setup_macs()
 		lan_mac=$(mtd_get_mac_text "mac" 4)
 		wan_mac=$(mtd_get_mac_text "mac" 24)
 		;;
+	dlink,dir-842-c1|\
```

Comment:
You are missing actual initialization of LAN/WAN interfaces & the switch in the top of the file.

Reply:
fixed, thanks! Sorry for the mistake